lib/google/gax/operation.rb

```diff
@@ -133,9 +133,20 @@ def initialize(grpc_op, client, result_type, metadata_type,
       #   The result of the operation. If it is an error a Google::Rpc::Status
```

Comment:
Blah, looks like I didn't fix my documentation in my last commit. 😞 

This should read:
```ruby
# If the operation is done, returns the response.
# If the operation response is an error, the error will be returned.
# Otherwise returns nil.
```